### DISCUSSION TITLE
404页面的配置文件在什么地方？

### DISCUSSION THREAD

**Discussion by OOAAHH** [pengzhanbo/vuepress-theme-plume#328] [Q&A] [answered]
我在我生成的模版文件中找不到对应的配置文件，我想自定义这个界面，变成“宝贝回家”之类的页面，请问我该从哪里开始？

**Comment by pengzhanbo** [comment, marked as answer]
[主题配置 > 404 Not Found](https://theme-plume.vuejs.press/config/basic/#notfound)

或者通过 [布局插槽](https://theme-plume.vuejs.press/guide/layout-slots/) 进行自定义。

**Reply by OOAAHH** [reply]
谢谢大佬，是我忽略了404还叫notfound
